packages/components/src/entities/SampleHeader.tsx

```diff
@@ -35,7 +35,12 @@ import { PrintLabelsModal } from '../internal/components/labels/PrintLabelsModal
 
 import { invalidateLineageResults } from '../internal/components/lineage/actions';
 
-import { isAllProductFoldersFilteringEnabled, isAssayEnabled, isWorkflowEnabled } from '../internal/app/utils';
+import {
+    hasProductProjects,
```

Comment:
is `hasProductProjects` used in this component? I see it was added to the import here, but don't see it in the change below.